sdk/eventhub/event-hubs/src/eventDataAdapter.ts

```diff
@@ -66,6 +66,11 @@ export interface EventDataAdapterParameters {
   properties?: {
     [key: string]: any;
   };
+  /**
+   * The sequence number the event was published with
+   * when idempotent partitions are enabled.
+   */
+  readonly publishedSequenceNumber?: number;
```

Comment:
Thanks for updating the adapter too!

Reply:
I ended up removing this from public surface. The update is reverted.